certificate-service/devops/dev/config/application-dev.properties

```diff
@@ -3,4 +3,15 @@ database.name=certificate_service
 database.server=localhost
 database.password=cs
 database.username=cs
-server.port=18030
\ No newline at end of file
+server.port=18030
+spring.activemq.broker-url=tcp://localhost:61616\
+     ?jms.nonBlockingRedelivery=true\
+     &jms.redeliveryPolicy.maximumRedeliveries=3\
+     &jms.redeliveryPolicy.maximumRedeliveryDelay=6000\
+     &jms.redeliveryPolicy.initialRedeliveryDelay=4000\
+     &jms.redeliveryPolicy.useExponentialBackOff=true\
+     &jms.redeliveryPolicy.backOffMultiplier=2
+spring.activemq.user=activemqUser
+spring.activemq.password=activemqPassword
+spring.activemq.pool.enabled=true
+spring.activemq.pool.max-connections=50
```

Comment:
Check if the 2 above props should be in application.properties instead. Perhaps have a look at Intygstjanst.

Reply:
If we use CachedConnectionFactory these are not needed correct?